packages/providers/mgt-msal2-provider/src/Msal2Provider.ts

```diff
@@ -223,11 +252,12 @@ export class Msal2Provider extends IProvider {
       this._loginType = typeof config.loginType !== 'undefined' ? config.loginType : LoginType.Redirect;
       this._loginHint = typeof config.loginHint !== 'undefined' ? config.loginHint : null;
       this._sid = typeof config.sid !== 'undefined' ? config.sid : null;
-      this._domainHint = typeof config.domain_hint !== 'undefined' ? config.domain_hint : null;
+      this._domainHint = typeof config.domainHint !== 'undefined' ? config.domainHint : null;
       this.scopes = typeof config.scopes !== 'undefined' ? config.scopes : ['user.read'];
       this._publicClientApplication = new PublicClientApplication(this.ms_config);
       this.isMultipleAccountDisabled =
         typeof config.isMultiAccountDisabled !== 'undefined' ? config.isMultiAccountDisabled : false;
+      this._prompt = typeof config.prompt !== 'undefined' ? config.prompt : PromptType.SELECT_ACCOUNT;
```

Comment:
I don't see `this._prompt` being used anywhere - shouldn't it be?

Reply:
Oh no, great catch. I think it got undone in one of my previous commits. Let me add it back

Reply:
Added and tested it to make sure :)

Reply:
Added!